<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BankSupervision</class>
 <widget class="QMainWindow" name="BankSupervision">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BankSupervision</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="GB1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>14</number>
       </property>
       <item>
        <widget class="QPushButton" name="CA">
         <property name="text">
          <string>Chiffre d'affaire</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Volume">
         <property name="text">
          <string>Volume</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Pourcentage">
         <property name="text">
          <string>Pourcentage</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="datedebut">
         <property name="dateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2000</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
         <property name="maximumDateTime">
          <datetime>
           <hour>23</hour>
           <minute>59</minute>
           <second>59</second>
           <year>9999</year>
           <month>12</month>
           <day>31</day>
          </datetime>
         </property>
         <property name="displayFormat">
          <string>dd-MMM-yyyy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateFin">
         <property name="dateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2020</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
         <property name="displayFormat">
          <string>dd-MMM-yyyy</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="Agences">
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QRadioButton" name="AgenceParis">
        <property name="text">
         <string>Agence Paris</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="AgenceBordeaux">
        <property name="text">
         <string>Agence Bordeaux</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="AgenceLyon">
        <property name="text">
         <string>Agence Lyon</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="AgenceStrasbourg">
        <property name="text">
         <string>Agence Strasbourg</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="AgenceToulouse">
        <property name="text">
         <string>Agence Toulouse</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QTabWidget" name="AgenceVue">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <widget class="QWidget" name="CanalG">
         <attribute name="title">
          <string>Canal Général</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QGridLayout" name="CGeneralLayout"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="CanalB">
         <attribute name="title">
          <string>Canal Bancaire</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QGridLayout" name="CBancaireLayout"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="CanalA">
         <attribute name="title">
          <string>Canal Assurance</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QGridLayout" name="CAssuranceLayout"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="CanalS">
         <attribute name="title">
          <string>Canal Boursier</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QGridLayout" name="CBoursierLayout"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>789</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
